src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    display: flex;
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
```

Comment:
Consider using styles for modifiers from the previous task

Reply:
> Consider using styles for modifiers from the previous task

Didn`t get it. What exactly should I do? I already used all the code from the task with stars (I mean, the part needed).

Reply:
> > Consider using styles for modifiers from the previous task
> 
> Didn`t get it. What exactly should I do? I already used all the code from the task with stars (I mean, the part needed).

but you need to reuse all styles from that task